demisto_sdk/commands/validate/validators/BA_validators/BA127_is_valid_context_path_depth_modified.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+from typing import Dict, Iterable, List, Set, Union
+
+from demisto_sdk.commands.common.constants import XSOAR_SUPPORT, GitStatuses
+from demisto_sdk.commands.content_graph.objects.base_content import BaseContent
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.validate.validators.BA_validators.BA127_is_valid_context_path_depth import (
+    IsValidContextPathDepthValidator,
+)
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script]
+
+
+class IsValidContextPathDepthModifiedValidatorModified(
+    IsValidContextPathDepthValidator, BaseValidator[ContentTypes]
+):
+    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.RENAMED]
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        results: List[ValidationResult] = []
+        for content_item in content_items:
+            if content_item.support != XSOAR_SUPPORT:
+                continue
+            old_content_item = content_item.old_base_content_object
+            if isinstance(content_item, Script):
+                invalid_paths = self.check_for_script_invalid_paths(
+                    old_content_item, content_item
+                )
+                if invalid_paths:
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message.format(
+                                "script", content_item.name, invalid_paths
+                            ),
+                            content_object=content_item,
+                        )
+                    )
+            else:
+                invalid_paths_dict = self.check_integration_invalid_paths(
+                    old_content_item, content_item
+                )
+                if invalid_paths_dict:
+                    message = ""
+                    for command, outputs in invalid_paths_dict.items():
+                        message += (
+                            self.error_message.format("command", command, outputs)
+                            + "\n"
+                        )
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=message,
+                            content_object=content_item,
+                        )
+                    )
```

Comment:
Still not sure why this would not work
```suggestion
                if isinstance(content_item, Script):
                    invalid_paths = self.check_for_script_invalid_paths(old_content_item, content_item)
                    if invalid_paths:
                        message = self.error_message.format("script", content_item.name, invalid_paths)
                else:
                    invalid_paths_dict = self.check_integration_invalid_paths(old_content_item, content_item)
                    if invalid_paths_dict:
                        message = ""
                        for command, outputs in invalid_paths_dict.items():
                            message += (
                                    self.error_message.format("command", command, outputs)
                                    + "\n"
                            )
                    results.append(
                        ValidationResult(
                            validator=self,
                            message=message,
                            content_object=content_item,
                        )
                    )
```